omego/external.py

```diff
@@ -176,13 +176,15 @@ def setup_previous_omero_env(self, olddir, savevarsfile):
         env = self.get_environment(savevarsfile)
 
         def addpath(varname, p):
-            if not os.path.exists(p):
-                raise Exception("%s does not exist!" % p)
-            current = env.get(varname)
-            if current:
-                env[varname] = p + os.pathsep + current
+            current = env.get(varname, "")
```

Comment:
Should we still raise an exception for https://github.com/ome/omego/pull/126/files#diff-05fc5084c51648b417f64d4924276946R193. In that case, some form of  `def addpath(varname, p, fail_if_missing=True):` signature should do the work.

Reply:
I don't think it matters, I'm not sure we rely on saving the environment any more since we've got a single Ice version.